GenumKit/Stencil/Filters.swift

```diff
@@ -80,12 +80,32 @@ struct StringFilters {
 
 struct ArrayFilters {
   static func join(value: Any?) throws -> Any? {
-    guard let array = value as? [Any] else { throw FilterError.InvalidInputType }
-    let strings = array.flatMap { $0 as? String }
-    guard array.count == strings.count else { throw FilterError.InvalidInputType }
+    let strings: [String]
+    if let array = value as? [Any] {
+      strings = array.flatMap { $0 as? String }
+      guard array.count == strings.count else { throw FilterError.InvalidInputType }
+    } else if let array = value as? [String] {
+      strings = array
+    } else {
+      throw FilterError.InvalidInputType
+    }
 
     return strings.joinWithSeparator(", ")
   }
+
+  static func joinAnd(value: Any?) throws -> Any? {
```

Comment:
I've implemented this in master now, you can find an example of implementing a filter with arguments at https://github.com/kylef/Stencil/commit/dfd57e957125e5178bf4e9d401fa892992783306.

It's going to be a few days until I cut a release with this though, there are other things I want to get into this release.